.vib/etcd/goss/goss.yaml

```diff
@@ -0,0 +1,34 @@
+http:
+  http://etcd:{{ .Vars.service.ports.peer }}:
```

Comment:
Can we change this here to `/version`? This will allow checking for a status of `200` instead of `404`, which is kind of weird.